app/views/_includes/application/details.njk

```diff
@@ -17,7 +17,7 @@
       items: [
         {
           href: '/application/' + application.id + '/cycle/edit',
-          text: "Change",
+          text: "Defer offer",
```

Comment:
We considered this but it is inconsistent with all the other change links we use and it also means the link text would have to change to ‘Undefer’ (or similar).

Users have the burden of rediscovering it.

That said I think we need to keep an eye on research to grasp people's mental model.

Reply:
Ah OK